lib/hijack/http.js

```diff
@@ -92,3 +98,43 @@ if (Package['fetch']) {
     }
   };
 }
+
+const { Agent } = require('_http_agent');
+
+const originalAddRequest = Agent.prototype.addRequest;
+
+Agent.prototype.addRequest = function () {
+  const request = arguments[0];
+  const options = arguments[1];
+  const port = options?.port ?? arguments[2];
```

Comment:
You can simplify this with `function (request, options, port)`, and still use `arguments` when calling `originalAddRequest`